darshan-util/pydarshan/darshan/experimental/plots/plot_access_histogram.py

```diff
@@ -26,41 +23,20 @@ def plot_access_histogram(report, mod, ax=None):
     if 'mod_agg_iohist' in dir(report):
         report.mod_agg_iohist(mod)
     else:
-        print("Can not create summary, mod_agg_iohist aggregator is not registered with the report class.")
+        print(
+            "Cannot create summary, mod_agg_iohist aggregator"
+            "is not registered with the report class."
+        )
```

Comment:
I wouldn't spend much energy changing line lengths unless we start enforcing that in a linter/CI (probably not a good use of time for now). Even if the editor does it automatically I still have to parse it myself when reading a diff.

Also, as the CI shows, this line isn't covered by tests, and probably should be something other than a plain `print()`.

Reply:
Okay, I've reverted the line length changes. 

Yeah, it should probably raise an error. I think if this is triggered it ends up raising an error anyways, so might as well. The same thing is present in `plot_opcounts.py`, so I'll open an issue for both cases.

Reply:
> Also, as the CI shows, this line isn't covered by tests, and probably should be something other than a plain print().

I've opened issue #588 to address this.